scripts/exrrfs_ic.sh

```diff
@@ -3,10 +3,10 @@ declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LI
 set -x
 cpreq=${cpreq:-cpreq}
 if [[ -z "${ENS_INDEX}" ]]; then
-  prefixin=${IC_PREFIX:-IC_PREFIX_not_defined}
+  prefixin=${EXTRN_MDL_SOURCE:-IC_PREFIX_not_defined}
   ensindexstr=""
 else
-  prefixin=${ENS_IC_PREFIX:-ENS_IC_PREFIX_not_defined}
+  prefixin=${EXTRN_MDL_SOURCE:-ENS_IC_PREFIX_not_defined}
```

Comment:
Line 6 and 9:
The part after `-` needs to modified to `EXTRN_MDL_SOURCE_not_defined` to match the new situation.

BTW, I don't use "" because I found that when we check the log file, the "not_defined" text will be more verbose to indicate what's wrong.

Reply:
changed